templates/Entry/entries.html.twig

```diff
@@ -18,6 +21,28 @@
     {% include "Entry/_title.html.twig" with {'filter': filter} %}
 {% endblock %}
 
+{% block nav_panel_extra_actions %}
+    <li>
+        <a class="waves-effect tooltipped js-random-action" data-position="bottom" data-delay="50" data-tooltip="{{ 'menu.top.random_entry'|trans }}" href="{{ path('random_entry', {'type': active_route}) }}">
+            <i class="material-icons">casino</i>
+        </a>
+    </li>
```

Comment:
wrapping check `active_route and is_granted('LIST_ENTRIES')` removed as now the button is only on the entries page and that condition is always true there